components/brave_wallet/renderer/brave_wallet_js_handler.h

```diff
@@ -71,7 +71,6 @@ class BraveWalletJSHandler : public mojom::EventsListener {
                    const int http_code,
                    const std::string& response,
                    const base::flat_map<std::string, std::string>& headers);
-
```

Comment:
Keep this.

Reply:
Fixed and interactive rebase squashed it into d17f0bdbff03fe6e299ea43ee030540a4ee08eca